blackberggroup/VA-iEX
# Fix: 7105768548 - landing page review

## What type of PR is this? (check all applicable)


Review changes from Will for the new IEX 24 page

## Related Tickets & Documents
- Monday Task:
 https://blackberg-group.monday.com/boards/1931719392/pulses/7169924800

## QA Instructions, Screenshots, Recordings

Review the following changes:

HERO
- Double check hero Register button size (padding x & y). Mockup shows it larger than other buttons on the page.
- Extra space after the word "Hotel". Since it's a link, it shows up more definite. 
- Button text is white, should be black. 

SHARK TANK
- Experiencing odd background svg jumping when scrolling down the page.

GENERAL
- ALL buttons should have a glow. Please see mockup. View right panel under "Effects".
- All images need more descriptive Alt text, "image-1" is not descriptive enough.
https://accessibility.huit.harvard.edu/describe-content-images#:~:text=Alt%20Text%20Best%20Practices,%E2%80%9Dor%20%E2%80%9Cpicture%20of.%E2%80%9D
- Registration Link and video url are found in the doc file shared in Slack on July 29th.
https://vhaohilfy24.slack.com/archives/C04RZ0P0U3U/p1722254476290599

KEYNOTE SPEAKERS
- Speaker name is incorrect font-family. Please view mockup. They should be using a heading instead of a div wrapper (SEO).

LOOK BACK AT iEX 2023
- Title html should be capitalized text with a class, "uppercase" added to the element wrapper. Not all uppercase html text.
- Video image has black bars on the left and right. Please view mockup for proper image layout.
- Add link to video, found in word doc.

ABOUT
- UPDATE: Client wants PDF added to this section. Please see mockup and PDF is attached.


### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_


## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized 

## Changed Files in this Pull Request ##
- _site/assets/documents/iEX
 - 2024
 - Registration
 - One-Pager.pdf
 - assets/documents/iEX
 - 2024
 - Registration
 - One-Pager.pdf
 - _includes/iex-2024/about.html
 - _includes/iex-2024/hero.html
 - _site/assets/css/styles.css
 - _site/assets/sass/vha/components/button/_index.scss
 - _site/assets/sass/vha/pages/iex-2023/counter/_index.scss
 - _site/index.html
 - assets/css/styles.css
 - assets/sass/vha/components/button/_index.scss
 - assets/sass/vha/pages/iex-2023/counter/_index.scss
